src/procedural-macros.md

```diff
@@ -240,6 +240,26 @@ A helper attribute for a derive macro is declared by adding its identifier to th
 > }
 > ```
 
+r[macro.proc.derive.attributes.scope]
+Helper attributes are in scope after the macro that introduces them.
```

Comment:
Would it be possible to be more specific about what "after" means here? There's some start point and some end point, correct?

That is, there are different interpretations of what "after" means:
- Anything after that point to the end of the file.
- Only *inside* the attached item.
- Inside the attached item *and* inside the output of the derive.
- Inside the attached item *and* any outer attribute *after* the `derive`.

The last one is my understanding of what it covers. I'm not sure what the best way to phrase that is.